internal/luks2/metadata.go

```diff
@@ -360,20 +360,26 @@ func (n JsonNumber) Uint64() (uint64, error) {
 	return strconv.ParseUint(string(n), 10, 64)
 }
 
+// Requirements corresponds to the requirements object in the JSON metadata of
+// a LUKS2 volume.
```

Comment:
given that there's discrepancy between the official pdf and the code, this could use a reference to upstream repo: https://gitlab.com/cryptsetup/cryptsetup/-/blob/93bc8997eaf6168bd196c82cb3adbd28b8d2ac56/lib/luks2/luks2_json_metadata.c?page=2#L1065 unless there's a better place where metadata is defined

Reply:
I've added a comment now.